pkg/data/data_store.go

```diff
@@ -40,19 +41,38 @@ func (ds *DataStore) Get(order string, skip, take int, startDate, finishDate tim
 	}).Skip(skip).Sort(order).Limit(take).All(&data)
 	if err != nil {
 		ds.logger.Error(err)
+		return nil, err
 	}
 
 	if data == nil {
 		data = []Data{}
 	}
 
-	return data, err
+	return data, nil
```

Comment:
Please, mention this additional fix in the commit message.

Reply:
Done!